### PR TITLE
Add media query for desktop screen

### PR DESCRIPTION
**Pull request by abdallahmalima**
### In this part I:

- Add media query CSS for desktop screen
- Adjust all layouts to be responsive on both mobile and desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -317,3 +344,224 @@ span {
...
+    padding-left: 0%;
+    flex-direction: row;
+    position: static;
+  }
+
+  .about-p1 button {
+    width: 35%;
+  }
+
+  .tech-list-f {
```
Comment (jasonfelice) [inline style.css:516]: - [ ] Kindly make sure the tech list looks like the Figma design.
 - Please change padding, fonts, and color so it looks like the Figma Design.

## Your Design:
image

## Figma Design:
image

File: style.css [hunk 2]
```diff
@@ -317,3 +344,224 @@ span {
...
+    padding-right: 260px;
+    padding-top: 40px;
+    background-image: url("images/header_bg.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: 45px 25px !important;
+    background-size: 95%;
+  }
+
+  .social-links {
```
Comment (jasonfelice) [inline style.css:375]: ## Social Links
- [ ] Please increase the gap to `21px` to follow the design guidelines. 😄 
- [ ] They'll look better if you give them fixed width (`140px`).

File: style.css [hunk 3]
```diff
@@ -317,3 +344,224 @@ span {
...
+  .language-list {
+    display: flex;
+    width: 55%;
+    justify-content: space-between;
+    padding-left: 0%;
+    flex-direction: row;
+    position: static;
+  }
+
+  .about-p1 button {
```
Comment (jasonfelice) [inline style.css:512]: - [ ] Kindly follow the Figma design for this button.

## Your Design:
image

## Figma Design:
image

File: index.html [hunk 4]
```diff
@@ -233,7 +250,7 @@ <h1 class="header-title text-center">Contact me</h1>
                    placeholder="Enter your name here" 
                    value="Abdallah" required>
             <input class="input txt-input" type="email" name="email" placeholder="Enter your email here" value="abdallahantony55.aa@gmail.com" required>
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly remove the value attribute from the input elements. They are not necessary for this case.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @abdallahmalima, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No linter errors✔️
GitHub flow followed✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @abdallahmalima 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdallahmalima** [comment]
thanks

**Pull request merged**